.prettierignore

```diff
@@ -0,0 +1,4 @@
+dist
+node_modules
```

Comment:
Why do you need this? We limit running prettier on `{src,stories,.storybook}`, right?

Reply:
I had a problem with .DS_Store that why i create this file
But I will remove that lines indeed